.github/ISSUE_TEMPLATE/mentoring_request.md

```diff
@@ -1,22 +0,0 @@
----
```

Comment:
we could potentially change this to a redirect (via the `config.yml`) to the significantly altered mentoring docs that I mentioned above